tests/resource/test_resource_result_parser.py

```diff
@@ -63,6 +63,19 @@ def test_resource_admin_throws_error_on_add_existing_profile(
             + f"'{profile_name}' already exists as a profile in the '{class_name}' class.",
         )
 
+    def test_resource_admin_avoids_error_on_add_covered_profile(
```

Comment:
What does `covered_profile` mean?

Reply:
The intent is to mean a profile "covered" (or protected) by a more generic one. In this case, TEST* already exists which would appear on RLIST TESTING as TEST* does "cover" the TESTING resource (as well as anything else that starts with TEST and has no lower qualifiers). This is a word I have thrown around on L2 support a lot, but I'm not sure if it came from documentation/standards or just my understanding of how profiles work.

Reply:
I see this is for testing when there is already a more generic profile protecting the resource. Should there be a similar test for data set admin or do we think this is good enough?

Reply:
The same edge case exists, but if I am not mistaken it requires driving the GENERIC keyword on LISTDSD (which maps to the base:generic trait on dataset admin command) so it is not as likely to appear especially with the lack of subfunctions or settrs. I think it should probably get a test case for completeness, but I wasn't rushing to get it into the beta with the other features. I will try to add it today before this branch gets merged.